toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -316,12 +317,14 @@ func (b *LiveOSIsoBuilder) prepareRootfsForDracut(writeableRootfsDir string) err
 // outputs:
 // - returns a SavedConfigs objects with the new merged values.
 func updateSavedConfigs(savedConfigsFilePath string, newKernelArgs imagecustomizerapi.KernelExtraArguments,
```

Comment:
nit:

```golang
func updateSavedConfigs(savedConfigsFilePath string, newKernelArgs imagecustomizerapi.KernelExtraArguments,
	newPxeIsoImageBaseUrl string, newPxeIsoImageFileUrl string, newDracutPackageInfo *PackageVersionInformation,
	newSELinuxPackageInfo *PackageVersionInformation,
) (updatedSavedConfigs *SavedConfigs, err error) {
```

This makes it easier to distinguish between the function signature and the first statement.

Reply:
pending

Reply:
fixed.